### ISSUE TITLE
Bug: Groups of simultanous keys kills repeat rate

### ISSUE THREAD

**Issue by kbilsted** [houmain/keymapper#161]
Not sure if this is a bug. But here goes

Using the configuration

```
AltLeft{H}	        >> ArrowLeft
AltLeft{J}	        >> ArrowDown
```

everything works nicely. I can hold down the keys and they repeat like the cursor keys do.

When I add a group selector

```
(J K) 		>> TO_NORMAL
```

then the repeat of `J` is no longer firing

**Comment by houmain** [comment]
No, this is not a bug. It waits whether the `K` follows or not. It starts repeating as soon as you also press the 'K'.
You could try `!AltLeft (J K) >> TO_NORMAL` or the recently added `?`, which makes it skip the mapping as long as it does not match completely: `? (J K) >> TO_NORMAL`

**Comment by kbilsted** [comment]
Great thanks for the quick reply. Would it be best practice to prefix `?` to all definitions? 
When wouldn't this be a good idea?

**Comment by kbilsted** [comment]
ok that was a bad idea using `?` it spins keymapper out of control using nearly all cpu and eating 2-3 MB ram a second

![image](https://github.com/user-attachments/assets/d6b1e2dd-b908-4f97-b4e3-3d4729a0ef7a)

**Comment by houmain** [comment]
Thanks, this should be fixed in 4.4.5.

>  Would it be best practice to prefix ? to all definitions?

While `? (J K)` immediately outputs J or K when one is pressed alone, `(J K)` does not. So it depends on your usecase.
